src/Flicking.ts

```diff
@@ -677,8 +697,7 @@ class Flicking extends Component {
       maintained.forEach(([, next]) => { viewport.updateCheckedIndexes({ min: next, max: next }); });
     }
     panelManager.replacePanels(newPanels, newClones);
-
-    !isCallSync && this.viewport.beforeSync();
+    this.flag = true;
```

Comment:
`this.flag` should have meaningful name.

Reply:
Oops I forgot to change that one.